src/dymaptic.GeoBlazor.Core/Components/Views/MapView.razor.cs

```diff
@@ -1845,14 +1956,43 @@ protected override async Task OnParametersSetAsync()
             }
         }
 
+        if (GraphicSerializationChunkSize is null)
+        {
+            int? chunkSetting = Configuration.GetValue<int?>("GraphicSerializationChunkSize");
+
+            if (chunkSetting is not null)
+            {
+                GraphicSerializationChunkSize = chunkSetting.Value;
+            }
+        }
+
         await UpdateView();
     }
+    
+    protected override async Task OnInitializedAsync()
+    {
+#if NET7_0_OR_GREATER
+        if (IsWebAssembly)
+        {
+#pragma warning disable CA1416
+            await JSHost.ImportAsync("arcGisJsInterop", "../_content/dymaptic.GeoBlazor.Core/js/arcGisJsInterop.js");
```

Comment:
will this URL be fixed for every blazor project? SHould it be configurable?

Reply:
Yes, it's fixed.  I can't recall exactly how or why there is a `../` in the path, but `_content/projectNamespace` is the standard path for Blazor libraries, which takes you to the `wwwroot` in that library, and then you append `js` which is the folder.